pages/services/cassandra/2.6.0-3.11.4/operations/index.md

```diff
@@ -0,0 +1,313 @@
+nodes, all other nodes must be restarted to be brought up to date regarding the ip address of the new seed node. This
+operation is performed automatically.
+
+For example if `node-0` needed to be replaced you would execute:
+
+```bash
+dcos {{ model.packageName }} --name=<service-name> pod replace node-0
+```
+
+which would result in a recovery plan like the following:
+
+```bash
+$ dcos {{ model.packageName }} --name=<service-name> plan show recovery
+recovery (IN_PROGRESS)
+└─ permanent-node-failure-recovery (IN_PROGRESS)
+   ├─ node-0:[server] (COMPLETE)
+   ├─ node-1:[server] (STARTING)
+   └─ node-2:[server] (PENDING)
+   ...
+```
+
+<p class="message--note"><strong>NOTE: </strong>Only the seed node is being placed on a new node, all other nodes are restarted in place with no loss of data.</p>
+
+
+# Back up and Restore
+
+## Backing Up to S3
+
+You can back up an entire cluster's data and schema to Amazon S3 using the `backup-s3` plan. This plan requires the following parameters to run:
+- `SNAPSHOT_NAME`: the name of this snapshot. Snapshots for individual nodes will be stored as S3 folders inside of a top level `snapshot` folder.
+- `CASSANDRA_KEYSPACES`: the {{ model.techShortName }} keyspaces to back up. The entire keyspace, as well as its schema, will be backed up for each keyspace specified.
+- `AWS_ACCESS_KEY_ID`: the access key ID for the AWS IAM user running this backup
+- `AWS_SECRET_ACCESS_KEY`: the secret access key for the AWS IAM user running this backup
+- `AWS_REGION`: the region of the S3 bucket being used to store this backup
+- `S3_BUCKET_NAME`: the name of the S3 bucket in which to store this backup
+- `HTTPS_PROXY`: specifications for the {{ model.TechName }} backup plan, taken from `config.yaml`.
+
+Make sure that you provision your nodes with enough disk space to perform a backup. {{ model.TechName }} backups are stored on disk before being uploaded to S3, and will take up as much space as the data currently in the tables, so you will need half of your total available space to be free to back up every keyspace at once.
+
+As noted in the documentation for the backup/restore strategy configuration option, it is possible to run transfers to S3 either in serial or in parallel, but care must be taken not to exceed any throughput limits you may have in your cluster. Throughput depends on a variety of factors, including uplink speed, proximity to region where the backups are being uploaded and downloaded, and the performance of the underlying storage infrastructure. You should perform periodic tests in your local environment to understand what you can expect from S3.
+
+You can configure whether snapshots are created and uploaded in serial (default) or in parallel. The serial backup/restore strategy is recommended.
+
+Before initiating the backup plan, it is recommended to reset the plan to its original state. This will ensure the backup plan runs in the correct sequence. You can reset the plan with:
```

Comment:
```suggestion
Before initiating the backup plan, it is recommended to reset the plan to its original state. This will ensure the backup plan runs in the correct sequence. You can reset the plan with the following command:
```